app/models/pusher.rb

```diff
@@ -126,10 +126,17 @@ def after_write
     Indexer.perform_later
     ReindexRubygemJob.perform_later(rubygem:)
     GemCachePurger.call(rubygem.name)
+    StoreVersionContentsJob.perform_later(version:) if ld_variation(key: "gemcutter.pusher.store_version_contents", default: false)
```

Comment:
https://app.launchdarkly.com/default/production/features/gemcutter.pusher.store_version_contents/targeting created the flag